### DISCUSSION TITLE
Hide location/date in photo cards

### DISCUSSION THREAD

**Discussion by dotfortun3-code** [photoprism/photoprism#3096] [Q&A] [answered]
Is is possible to hide the information about the photo when viewing the photos as large cards? I would like to be able to just have the photos without the location/date information, since I can just view the photo to get that information. I attached samples of what I am talking about for clarification

**What I Have**
![Screenshot 2023-01-12 134030](https://user-images.githubusercontent.com/24662541/212152078-499679c2-60e2-4cc1-a7c8-dec090c48ac2.png)

**What I would like to do**
![photocard-example](https://user-images.githubusercontent.com/24662541/212153342-282d34d7-2c30-4325-9443-6df73d448742.png)

**Comment by graciousgrey** [comment, marked as answer]
That's not possible right now, but as far as I remember we already have a feature request for it.

Cute pets by the way :heart:

**Reply by dotfortun3-code** [reply]
Alright, good to know it's on the feature request list. I love the app, keep up the great work!

Thanks! (Glad you liked the pets!)
